app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -745,19 +718,20 @@ public AptoideAccountManager getAccountManager() {
       final AccountFactory accountFactory = new AccountFactory();
 
       final AccountService accountService =
-          new V3AccountService(accountFactory, getDefaultClient(), getLongTimeoutClient(),
+          new AccountServiceV3(accountFactory, getDefaultClient(), getLongTimeoutClient(),
               WebService.getDefaultConverter(), getNonNullObjectMapper(),
               getDefaultSharedPreferences(), getExtraId(), getTokenInvalidator(),
               getAuthenticationPersistence(), getNoAuthenticationBodyInterceptorV3(),
-              getMultipartBodyInterceptor(), getBodyInterceptorWebV7(), getBodyInterceptorPoolV7());
+              getMultipartBodyInterceptor(), getBodyInterceptorPoolV7(),
+              getBodyInterceptorPoolV7());
```

Comment:
Two Interceptors of Pool type are being injected, one should be Web?

Reply:
Fixed.